Avoid tk +quartz bug (https://core.tcl-lang.org/tk/info/685ac30727)
by using (emulated) X clip region support (available in Tk Aqua as of 8.6.11)
Upstream-Status: Pending (Tktable is dormant)

--- generic/tkTable.c.orig	2008-11-14 17:43:35.000000000 -0600
+++ generic/tkTable.c	2020-08-29 10:44:15.000000000 -0500
@@ -1680,7 +1680,7 @@
  * rectangles.  Thus turn it off for all cases until clip rectangles
  * are known to be respected. [Bug 1805350]
  */
-#if 1 || defined(MAC_TCL) || defined(UNDER_CE) || (defined(WIN32) && defined(TCL_THREADS)) || defined(MAC_OSX_TK)
+#if !defined(MAC_OSX_TK)
 #define NO_XSETCLIP
 #endif
 /*
